tests/test_sqlite.py

```diff
@@ -27,3 +27,5 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.MD5PasswordHasher',
 ]
+
+GEOIP_PATH = '/usr/local/share/geoip'
```

Comment:
Please remove this from the default settings. What I do locally is to create a file called `test_sqlite3.py` that imports this file and has any extra settings I need. You can also add it to `.git/info/exclude`.